infrastructure/preview.tfvars

```diff
@@ -2,6 +2,7 @@ feature_fine_print = "true"
 feature_new_features_consent = "true"
```

Comment:
you can delete this file and any `sprod`, `saat` ones too pls

Reply:
done, please check

Reply:
still showing 🤔 missed git add maybe?